tortoise/verifying.go

```diff
@@ -88,12 +88,12 @@ func (v *verifying) verify(logger log.Log, lid types.LayerID) bool {
 	threshold := v.globalThreshold(v.Config, lid)
 	logger = logger.WithFields(
 		log.String("verifier", "verifying"),
-		log.Stringer("candidate layer", lid),
+		log.Stringer("candidate_layer", lid),
 		log.Stringer("margin", margin),
 		log.Stringer("uncounted", uncounted),
 		log.Stringer("total good weight", v.totalGoodWeight),
 		log.Stringer("good uncounted", layer.verifying.goodUncounted),
-		log.Stringer("global threshold", threshold),
+		log.Stringer("global_threshold", threshold),
```

Comment:
Field naming is inconsistent now - some use `_` and some - ` `.

Reply:
changed it so it is consistent with the full mode when inspecting logs

Reply:
so the goal here is that the field name is consistent btwn verifying.go and full.go
i've updated the fields here to use "_". will leave those "-" untouched in this pkg.